dev/release/02-source-test.rb

```diff
@@ -110,7 +110,14 @@ def test_vote
                      "?state=open" +
                      "&head=apache:release-#{@release_version}-rc0")
     verify_pr_url = nil
-    verify_prs.open("Accept" => "application/vnd.github+json") do |response|
+    headers = {
+      "Accept""Accept" => "application/vnd.github+json",
```

Comment:
I don't have much idea on Ruby but is this double `"Accept""Accept"` correct?

Reply:
Oh, sorry... It's garbage...